pdns/syncres.cc

```diff
@@ -369,9 +369,12 @@ int SyncRes::asyncresolveWrapper(const ComboAddress& ip, bool ednsMANDATORY, con
     
     if(mode==EDNSStatus::NOEDNS) {
       g_stats.noEdnsOutQueries++;
+      if (ednsMANDATORY) {
```

Comment:
We should probably rename `ednsMANDATORY` to something like `wantsDNSSEC`, since it's not mandatory anymore.

Reply:
agreed! maybe `wantsEDNS` as it can be used for other things than DNSSEC (EDNSCookies etc.)